packages/plugins/docker/src/lib/publish.ts

```diff
@@ -17,14 +17,21 @@ export async function publish(
     name,
   } = pluginConfig;
 
+  // TODO Provide configuration for selecting how to handle tags on non-default channels.
+
+  // Only publish additional tags for the default channel.
+  const publishAdditionalTags = channel === null;
```

Comment:
So there should be property on the context where you can test if it is a prerelease.  You can see this in the @semantic-release-plus/semantic-release repo you can see this in https://github.com/semantic-release-plus/semantic-release/blob/master/lib/get-last-release.js#L33 and a couple other places.